goalie/interpolation.py

```diff
@@ -154,3 +172,21 @@ def _project_adjoint(target_b, source_b, **kwargs):
 
     # Map back to a Cofunction
     return function2cofunction(source_b)
+
+
+@PETSc.Log.EventDecorator()
```

Comment:
Drop the event decorator for private functions like this because (a) the user won't know what the private function is from the API docs and (b) its runtime is likely to be negligible anyway.

Reply:
Thanks! Fixed in 76606e87a42b7d9a37456f1af5ed1d4adde7e4a5